src/components/Note/Note.js

```diff
@@ -22,26 +22,32 @@ import { Text } from 'react-native';
 import styled from 'styled-components/native';
 import Emoji from 'react-native-emoji';
 import { BaseText } from 'components/Typography';
-import { fontStyles } from 'utils/variables';
+import { fontStyles, spacing } from 'utils/variables';
 import { themedColors } from 'utils/themes';
 
 type Props = {
   note: string | React.Node,
   emoji?: string,
   containerStyle?: Object,
+  childrenOnRight?: React.Node,
```

Comment:
There aren't any other `children` for now, so I guess you can just name this `children` :)

Reply:
Yeah, was thinking that we might wanna add children in footer, so I've named this prop according to position. Since those are the first ones I'll rename them to "children".